dts/bindings/flash_controller/nuvoton,npcx-fiu-nor.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2023 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    The SPI NOR flash devices accessed by Nuvoton Flash Interface Unit (FIU).
+
+    Representation of a SPI NOR flash on a qspi bus looks like:
+
+      int_flash: w25q40@0 {
+            compatible ="nuvoton,npcx-fiu-nor";
+            size = <0x400000>; /* 4194304 bits = 512K Bytes */
```

Comment:
Same as the other comment, this could be just `size = <DT_SIZE_K(512 * 8)>;` and no need for a comment

Reply:
Done.